src/models/timeslot.db.model.ts

```diff
@@ -1,5 +1,5 @@
 import { LessonDbModel } from './lesson.db.model';
-import RoomDbModel from './room.db.model';
+import RoomDbModel from './location.db.model';
```

Comment:
if we already renaming to locations then rename the dbmodel as well to LocationDbModel

Reply:
yes you right